packages/react-components/src/components/ChatMessage/ChatMessageComponent.tsx

```diff
@@ -22,6 +22,7 @@ type ChatMessageComponentProps = {
    * Setting to false will mean they are on a new line inside the editable chat message.
    */
   inlineAcceptRejectEditButtons: boolean;
+  participantCountNotIncludingSelf?: number;
```

Comment:
I dont think this is better but what about

"numOtherParticipants"

Reply:
`remoteParticipantsCount` would be consistent with naming elsewhere in our library (and headless SDKs)

Reply:
Aha, I like the idea of 'remote' which means not the local user

Reply:
Changed name to remoteParticipantsCount